src/elements/content-sidebar/stories/tests/MetadataSidebarRedesign-visual.stories.tsx

```diff
@@ -50,3 +52,62 @@ export default {
         ...defaultVisualConfig.parameters,
     },
 };
+
+export const MetadataInstanceEditorWithCustomMetadata: StoryObj<typeof MetadataSidebarRedesign> = {
+    play: async ({ canvasElement }) => {
+        const canvas = within(canvasElement);
+        await canvas.findByRole('heading', { name: 'Metadata' });
```

Comment:
can we wrap this is an `expect`, the result of this will show up in the interaction tab and will help wit debugging if this fails

Reply:
added